backend/queue/queue.go

```diff
@@ -46,9 +45,9 @@ func (e EtcdGetter) GetQueue(path ...string) types.Queue {
 
 // Queue is a durable FIFO queue that is backed by etcd.
```

Comment:
We may want to qualify "durable" a bit here.

Reply:
You're right, it is no longer durable.